pkg/vm/engine/tae/tables/txnentries/mergeobjects.go

```diff
@@ -95,6 +98,10 @@ func NewMergeObjectsEntry(
 func (entry *mergeObjectsEntry) prepareTransferPage() {
```

Comment:
I think it should return `err error` now and the caller need to handle error scenario